src/database/models/reviews.js

```diff
@@ -35,6 +35,10 @@ module.exports = (sequelize, DataTypes) => {
 				type: DataTypes.STRING,
 				primaryKey: true,
 			},
+			notes: {
+				allowNull: true,
+				type: DataTypes.STRING,
+			},
```

Comment:
@adithyadinesh0412 update the index as per my PR

Reply:
@adithyadinesh0412  update the index in model as per my PR. I guess org id also will come